packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v0.177.0
+------------------------------
+*September 1, 2021*
+
+### Changed
+- Gelocation warning to error for more details in Sentry
+
```

Comment:
Minor, should be an extra line break here :)

Reply:
Added